services/web/server/src/simcore_service_webserver/workspaces/_workspaces_db.py

```diff
@@ -217,18 +228,20 @@ async def update_workspace(
             )
             .returning(*_SELECTION_ARGS)
         )
-        row = await result.first()
+        row = result.first()
```

Comment:
Finally this one as well. I would expect this one to be await maybe the `execute` above should be a `stream`?

Reply:
@GitHK I do not really see the point in having that with requests that returns just an element.

All the research I have done indicates that `stream()` is preferable **when working with large datasets that can overwhelm memory** if loaded all at once, as it delivers rows incrementally through server-side cursors, enabling row-by-row processing without high memory consumption.

Moreover, in our case we do not even iterate those rows downstream but rather store **all** of them in memory before returning them.  Yes, at least we do not block while doing ... that is why i agreed in using it in the list operations. 

My point is that, instead of relying on "the theory", I would rather do some measurements to identify what is actually needed in which case and then adopt the best strategy on each situation.  That said, can you suggest a benchmark in which we can really measure that `await result.first()` is having a comparable effect with `result.first()`?